src/button/Button.tsx

```diff
@@ -74,12 +74,7 @@ function Button({
       disabled={isButtonDisabled}
       aria-label={ariaLabel}>
       {shouldDisplaySpinner ? (
-        <Spinner
-          customClassName={classNames("button__spinner", {
-            [`${customClassName}__spinner`]: customClassName
-          })}
-          aria-label={"Button spinner visible. Button inactivated."}
-        />
+        <Spinner aria-label={"Button spinner visible. Button inactivated."} />
```

Comment:
Should we keep `button__spinner` className?

Reply:
I think it seems we don't need it. But we can add `customClassName` prop again and we can pass that className.

Reply:
Yeah, let's add `customClassName` and pass that. We definitely gonna need `customClassName` because we're using the Spinner outside of the toolkit e.g. `RouteLoading`